irm/api/Snippets/Snippets.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.7.0" />
+    <PackageReference Include="xunit" Version="2.3.1" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.3.1" />
+    <DotNetCliToolReference Include="dotnet-xunit" Version="2.3.1" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="/usr/local/google/home/rennie/gitrepos/google-cloud-dotnet/apis/Google.Cloud.Irm.V1Alpha2/Google.Cloud.Irm.V1Alpha2/Google.Cloud.Irm.V1Alpha2.csproj" />
```

Comment:
Can you fix it at least to relative path?

Reply:
Done.